README.md

```diff
@@ -11,6 +11,8 @@ kube-consul-register watches Kubernetes events and converts information about PO
         time in seconds, what period of time will be done cleaning of inactive services (default 30m0s)
   -configmap string
         name of the ConfigMap that containes the custom configuration to use (default "default/kube-consul-register-config")
+  -secret string
+        name of the Secret that containts the consul token, key must be consul_token
```

Comment:
I assume that syntax for a secret here is `namespace/secret_name`, can you add an example in the description to keep it clear?